index.js

```diff
@@ -1 +1 @@
-module.exports = require('lib/index');
+module.exports = require('./lib/index');
```

Comment:
Given that you have a `main` in the `package.json` this file is superflous, but I used it during testing, so I found the glitch.